_posts/2020-03-16-dask-usage-at-blue-yonder.markdown

```diff
@@ -1,131 +1,131 @@
-means it is really easy to commission or decommission Dask clusters and to add or remove workers to/from
-an existing Dask cluster. At the moment, we are starting a migration from Mesos/Aurora towards 
-[Kubernetes](https://kubernetes.io/)
-
-Running on top of a system like Mesos or Kubernetes provides us with resilience since a failing worker 
-(for instance, as result of a failing hardware node)
-can simply be restarted on another node of the system. 
-
-Running 500 Dask clusters requires a lot of hardware. We have put two measures in place to improve
-the utilization of hardware resources: oversubscription and autoscaling.
-
-### Oversubscription
-
-[Oversubscription](http://mesos.apache.org/documentation/latest/oversubscription/) is a feature of Mesos
-that allows to allocate more resources than physically present to services running on the system.
-This is based on the assumption that not all services exhaust all of their allocated resources at the same time.
-If the assumption is violated, classifying services into tiers allows to preempt less important services
-in favor of critical ones.
-We use this to re-purpose the resources allocated for production clusters but not utilized the whole time
- and use them for development and staging systems.
-
-### Autoscaling
-
-Autoscaling is a mechanism we implemented to dynamically adapt the number of workers in a Dask cluster
-to the load on the cluster. For this, we added the ``desired_workers`` metric to Distributed, which exposes
-the degree of parallelism that a computation has and allows us to infer how much workers a cluster should
-ideally have. Based on this metric, as well as on the overall resources available and on fairness criteria
-(remember, we run a lot of Dask/Distributed clusters), we add or remove workers to our clusters.
-
-Restarting failed workers for resilience and adding/removing workers for autoscaling requires that the Distributed
-cluster can cope with workers being removed and/or new instances being added. While this is a supported use case, 
-it appeared to us that this seemed not a widely used feature. By making heavy use of it, we were able to uncover
-some bugs and to submit the corresponding fixes.
+  ---
+  layout: single
+  title:  "Dask Usage at Blue Yonder"
+  date:   2019-10-25 10:00:00 +0100
+  tags: technology python data-engineering
+  header:
+    overlay_image: assets/images/tech_gear_banner.jpg
+    overlay_filter: 0.2
+    show_overlay_excerpt: false
+  author: Andreas Merkel
+  author_profile: true
+  ---
+  # Dask Usage at Blue Yonder
+
+  Back in 2017, Blue Yonder started to look into 
+  [Dask/Distributed](https://distributed.dask.org) and how we can leverage it to power
+  our machine learning and data pipelines.
+  In 2019, we started to use it heavily in production for providing demand predicions,
+  recommended order quantities, and price proposals to our customers.
+  Time to take a a look at the way we are using Dask!
```

Comment:
I think this could be 
```
It's 2020, and we are using it heavily in production for providing demand predictions, recommending order quantities, and proposing prices to our customers. Let's have a look at how we are doing!
```